Sources/ChatkitBeamsTokenProvider.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+import PusherPlatform
+import PushNotifications
+
+private struct ChatkitBeamsToken: Decodable {
+    var token: String
```

Comment:
Does this need to be a `var` rather than a `let`?

Reply:
Fixed.